modules/deleting-cluster-aws.adoc

```diff
@@ -0,0 +1,21 @@
+// Module included in the following assemblies:
+//
+// * osd_install_access_delete_cluster/osd-deleting-a-cluster.adoc
+// * osd_getting_started/osd-getting-started.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="deleting-cluster_aws_{context}"]
+= Deleting your cluster
+
+You can delete your {product-title} cluster in {cluster-manager-first}.
+
```

Comment:
Suggest adding Prerequisites heading to both this module and the corresponding GCP one.
```suggestion
.Prerequisites
```